specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/workspaceRP.json

```diff
@@ -2368,15 +2427,34 @@
         "outboundRules": {
           "type": "object",
           "additionalProperties": {
+            "description": "Outbound Rule for the managed network of a machine learning workspace.",
             "$ref": "#/definitions/OutboundRule"
           },
           "x-nullable": true
         },
         "status": {
+          "description": "Status of the Provisioning for the managed network of a machine learning workspace.",
           "$ref": "#/definitions/ManagedNetworkProvisionStatus"
         }
       }
     },
+    "ManagedNetworkSettingsEx": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ManagedNetworkSettings"
+        }
+      ],
+      "properties": {
+        "changeableIsolationModes": {
```

Comment:
>changeableIsolationModes [](http://example.com/codeflow?start=9&length=24)

Just wondering...
Why call it 'changeable'? Do you just mean that its not readonly? Does that need to be part of the name?

Reply:
Confirming with this code owner, this is autogenerated based on code.

Reply:
will revert this one

Reply:
This is just how the code got implemented, I actually don't have much knowledge about this feature.